client/state/plugins/wporg/test/test-actions.js

```diff
@@ -0,0 +1,79 @@
+/**
+ * External dependencies
+ */
+import { assert } from 'chai';;
```

Comment:
we're not worried about the double semicolon - it's the invisible Unicode characters running some hard-to-read function between them that concerns us :wink: